salt/modules/cryptdev.py

```diff
@@ -0,0 +1,344 @@
+# -*- coding: utf-8 -*-
+'''
+Salt module to manage Unix cryptsetup jobs and the crypttab file
+'''
+
+# Import python libraries
+from __future__ import absolute_import
+import logging
+import os
+import re
+
+# Import salt libraries
+import salt.utils
+from salt.exceptions import CommandExecutionError
+
+# Import 3rd-party libs
+import salt.ext.six as six
+from salt.ext.six.moves import filter, zip  # pylint: disable=import-error,redefined-builtin
+
+# Set up logger
+log = logging.getLogger(__name__)
+
+# Define the module's virtual name
+__virtualname__ = 'cryptdev'
+
```

Comment:
As you mention in the PR body itself, we do need a `versionadded` here. It will be `Oxygen`